packages/@tests/smoke/web/public/index.js

```diff
@@ -1,10 +1,6 @@
-const fluence = globalThis.fluence;
+import { Fluence, callAquaFunction, randomStage } from "./js-client.min.js";
 
-const relay = {
-  multiaddr:
-    "/ip4/127.0.0.1/tcp/9991/ws/p2p/12D3KooWBM3SdXWqGaawQDGQ6JprtwswEg3FWGvGhmgmMez1vRbR",
-  peerId: "12D3KooWBM3SdXWqGaawQDGQ6JprtwswEg3FWGvGhmgmMez1vRbR",
-};
+const relay = randomStage();
```

Comment:
why stage? tests should run on the local network

Reply:
Thanks, you're right.